### PR TITLE
Instructions for syntax highlighting

### PR DESCRIPTION
**Pull request by sethladd**
Partially closes https://github.com/flutter/flutter/issues/8955

cc @mit-mit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +113,66 @@ next-page-title: "Create a Layout"
...
+highlighting. This section covers how to use syntax highlighting, and
+how to update our syntax highlighter for new languages.
+
+### Using syntax highlighting
+
+The easiest way to syntax highlight a block of code is to wrap
+it with `{% prettify _language_ %}`:
+
+```
+{% prettify dart %}
```
Comment (mit-mit) [inline README.md]: We recommend this over the "old" approach?

```
   ```dart
   class SomeCode {
     String name;
   }
   ```

File: README.md [hunk 2]
```diff
@@ -113,6 +113,66 @@ next-page-title: "Create a Layout"
...
+applies the necessary CSS classes for Prism to turn on syntax highlighting.
+You must specify a language for syntax highlighting to work properly.
+
+We don't currently have a good way to syntax highlight inline code. It may be
+possible, if you know Prism's CSS classes. "It's an exercise left up
+to the reader."
+
+### Supported languages
+
+This website can syntax highlight the following languages:
```
Comment (mit-mit) [inline README.md]: Can we list the actual values you need to enter in the markdown for `_language_` for each of these?

File: README.md (hunk 1 above)
Comment (sethladd) [inline README.md]: I'll mention you can do both. But, if you use triple-backtick, you lose some features such as inline highlighting and inline strike.

File: README.md [hunk 3]
```diff
@@ -113,6 +72,115 @@ next-page-title: "Create a Layout"
...
+* swift
+
+### Using syntax highlighting
+
+The easiest way to syntax highlight a block of code is to wrap
+it with triple backticks followed by the language.
+
+Here's an example:
+
+````
```
Comment (sethladd) [inline README.md]: these are intentional. Maybe there's a better way to escape the following triple backticks?

**Comment by sethladd** [comment]
@mit-mit ptal

**Pull request merged**